test/e2e/framework/test_context.go

```diff
@@ -26,12 +26,15 @@ import (
 )
 
 type TestContextType struct {
-	KubeConfig            string
-	KubeContext           string
-	KubeVolumeDir         string
-	CertDir               string
-	Host                  string
-	RepoRoot              string
+	KubeConfig    string
```

Comment:
Can we revert these changes? (aka set the padding back to the previous version?)